x-pack/plugins/uptime/server/lib/adapters/pings/elasticsearch_pings_adapter.ts

```diff
@@ -181,36 +178,24 @@ export class ElasticsearchPingsAdapter implements UMPingsAdapter {
               buckets: 25,
             },
             aggs: {
-              by_id: {
-                terms: {
-                  field: 'monitor.id',
-                  size: 200,
+              down: {
+                filter: {
+                  term: {
+                    'monitor.status': 'down',
+                  },
                 },
                 aggs: {
-                  status: {
-                    terms: {
-                      field: 'monitor.status',
-                      size: 2,
+                  bucket_count: {
+                    cardinality: {
```

Comment:
This will cost `25 * 1 * 3000` bytes (about `75 kB`) of heap, per shard. `3000` is the default `precision_threshold`.

You do need to do this type of aggregation to _estimate_ the number of _down_ monitors versus the number of `down` reports.